Formula/pinentry-mac.rb

```diff
@@ -3,7 +3,8 @@ class PinentryMac < Formula
   homepage "https://github.com/GPGTools/pinentry-mac"
   url "https://github.com/GPGTools/pinentry-mac/archive/v0.9.4.tar.gz"
   sha256 "037ebb010377d3a3879ae2a832cefc4513f5c397d7d887d7b86b4e5d9a628271"
-  license "GPL-2.0"
+  license "GPL-3.0-or-later"
```

Comment:
```suggestion
  license all_of: ["GPL-2.0-or-later", "GPL-3.0-or-later"]
```
Some of their source code seems to be `GPLv2+`: https://github.com/GPGTools/pinentry-mac/search?p=1&q=license

Reply:
Good catch 👍

Reply:
Amended the last commit ✔️